pages/visualizations/chaoss/contrib_drive_repeat.py

```diff
@@ -149,10 +150,21 @@ def create_drive_by_graph(repolist, timer_pings, contribs, view):
     logging.debug("CONTRIB_DRIVE_REPEAT_VIZ - START")
     start = time.perf_counter()
 
+    # create dataframe from record data
+    df = pd.DataFrame(results)
+
+    # test if there is data
+    if df.empty:
+        logging.debug("CONTRIB DRIVE REPEAT - NO DATA AVAILABLE")
+        return nodata_graph, False, dash.no_update
+
+    # convert to datetime objects with consistent column name
+    df["created_at"] = pd.to_datetime(df["created_at"], utc=True)
+    df.rename(columns={"created_at": "created"}, inplace=True)
+
     # graph on contribution subset
-    df_cont = pd.DataFrame(results)
-    contributors = df_cont["cntrb_id"][df_cont["rank"] == contribs].to_list()
-    df_cont_subset = pd.DataFrame(results)
+    contributors = df["cntrb_id"][df["rank"] == contribs].to_list()
+    df_cont_subset = pd.DataFrame(df)
```

Comment:
would you please comment these two lines, defining 'rank' and 'contribs'? I know what they mean but that might be confusing. Also, why are you creating a new df_cont_subet of df? (line 167?)

Reply:
df_cont_subset was just a new dataframe created to hold the subset of the entries that satisfy the condition. Not completely necessary to have it separate from df, just helpful when following along